templates/common/cloud-platform-alt-dns/files/coredns.yaml

```diff
@@ -0,0 +1,113 @@
+mode: 0644
+path: {{ if gt (len (cloudPlatformAPIIntLoadBalancerIPs .)) 0 -}} "/etc/kubernetes/manifests/coredns.yaml" {{ else }} "/etc/kubernetes/disabled-manifests/coredns.yaml" {{ end }}
+contents:
+  inline: |
+    kind: Pod
+    apiVersion: v1
+    metadata:
+      name: coredns
+      namespace: openshift-{{ cloudPlatformShortName . }}-infra
+      creationTimestamp:
+      deletionGracePeriodSeconds: 65
+      labels:
+        app: {{ cloudPlatformShortName . }}-infra-coredns
```

Comment:
@cybertron is the above comment regarding the name also applicable for the label "app"?

Reply:
I don't know if this matters. I'm not positive we ever actually look at this label.

However, I think if you give these all static names you could likely get rid of the cloudPlatformShortName entirely. That wouldn't need to block this though, it could easily be done as a followup.